site/docs/plugins/autoquote.md

```diff
@@ -0,0 +1,146 @@
+# Always Replying to Messages
+
+It is sometimes necessary to always send messages as replies, especially for bots that are meant to be used in groups.
+We usually do this by adding the `reply_to_message_id` parameter to the methods that send the message: `sendText`, `reply`, sendPhoto, replyWithPhoto and etc.
+However, if you're doing this for every single message, it can get messy and boring.
+
+This plugin sets the `reply_to_message_id` parameter to `ctx.msg.message_id` for `reply` and all `send*` methods that support it to make every message a reply to the message that triggered it.
+
+## Usage
+
+### In Specific Routes
+
+If you want all messages sent within a specific context (like a specific command), you can specifically apply the plugin to them:
+
+<CodeGroup>
+  <CodeGroupItem title="TypeScript" active>
+
+    ```ts
+    import { Bot } from "grammy";
+    import { addReplyParam } from "@roziscoding/grammy-autoquote";
+
+    const bot = new Bot("");
+
+    bot.command("demo", async (ctx) => {
+      ctx.api.config.use(addReplyParam(ctx));
+      await ctx.reply("Demo command!"); // this is going to quote the user's message
+    });
+
+    bot.start();
+    ```
+
+</CodeGroupItem>
+  <CodeGroupItem title="JavaScript">
+
+    ```ts
+    const { Bot } = require("grammy");
+    const { addReplyParam } = require("@roziscoding/grammy-autoquote");
+
+    const bot = new Bot("");
+
+    bot.command("demo", async (ctx) => {
+      ctx.api.config.use(addReplyParam(ctx));
+      await ctx.reply("Demo command!"); // this is going to quote the user's message
+    });
+
+    bot.start();
+    ```
+
+</CodeGroupItem>
+  <CodeGroupItem title="Deno">
+
+    ```ts
+    import { Bot } from "https://deno.land/x/grammy/mod.ts";
+    import { addReplyParam } from "https://deno.land/x/grammy_autoquote/mod.ts";
+
+    const bot = new Bot("");
+
+    bot.command("demo", async (ctx) => {
+      ctx.api.config.use(addReplyParam(ctx));
+      await ctx.reply("Demo command!"); // this is going to quote the user's message
+    });
+
+    bot.start();
+    ```
```

Comment:
````suggestion
```ts
import { Bot } from "https://deno.land/x/grammy/mod.ts";
import { addReplyParam } from "https://deno.land/x/grammy_autoquote/mod.ts";

const bot = new Bot("");

bot.command("demo", async (ctx) => {
  ctx.api.config.use(addReplyParam(ctx));
  await ctx.reply("Demo command!"); // this is going to quote the user's message
});

bot.start();
```
````